playwright.config.ts

```
@@ -1,38 +1,64 @@
+    ),
   },
 }));
 
+// Create ui projects for UI tests from exported test items
+let uiProjects: any[] = [];
+const exportedTestItemsPath = './tmp/test-items.json';
+if (existsSync(exportedTestItemsPath)) {
+  try {
+    const exportedData = JSON.parse(readFileSync(exportedTestItemsPath, 'utf-8'));
+    if (exportedData.testItems && Array.isArray(exportedData.testItems)) {
+      uiProjects = exportedData.testItems.map((itemData: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)